### PR TITLE
SEP-6: fix typos and update wording

### PR DESCRIPTION
**Pull request by marcelosalloum**
### What

Fix typos and update wording.

### CODE REVIEW

File: ecosystem/sep-0006.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ Request Parameters:
 
 Name | Type | Description
 -----|------|------------
-`asset_code` | string | The code of the asset the user wants to deposit with the anchor. The value passed must match one of the codes listed in the [/info](#info) response's deposit object.
+`asset_code` | string | The code of the off-chain asset the user wants to deposit with the anchor. The value passed must match one of the codes listed in the [/info](#info) response's deposit object.
```
Comment (JakeUrban) [inline ecosystem/sep-0006.md:134]: This is actually the on-chain asset, as specified in the `/info` description:

> The JSON object contains an entry for each Stellar asset that the anchor supports for SEP6 deposit and/or withdrawal.

File: ecosystem/sep-0006.md [hunk 2]
```diff
@@ -814,7 +814,7 @@ Request parameters:
 
 Name | Type | Description
 -----|------|------------
-`id` | string | (optional) The id of the transaction.
+`id` | (optional) string | The id of the transaction.
```
Comment (JakeUrban) [inline ecosystem/sep-0006.md:817]: I think we should keep this as-is, since it matches the syntax used in our other endpoints and SEPs

**Review by JakeUrban — CHANGES_REQUESTED** [review]
Most of these changes are great, I made 2 comments for minor adjustments and then it should be good to merge 👍

File: ecosystem/sep-0006.md (hunk 2 above)
Comment (marcelosalloum) [reply ecosystem/sep-0006.md]: Reverted the change in https://github.com/stellar/stellar-protocol/pull/972/commits/2672aba7e7d62e3cc830f7b5cc02b6c6326915bf.

File: ecosystem/sep-0006.md (hunk 1 above)
Comment (marcelosalloum) [reply ecosystem/sep-0006.md]: Reverted this change in https://github.com/stellar/stellar-protocol/pull/972/commits/2672aba7e7d62e3cc830f7b5cc02b6c6326915bf.

**Review by JakeUrban — APPROVED** [review]

**Pull request merged**